.buildkite/test/docker-compose.14.yml

```
@@ -31,7 +31,7 @@ services:
       - TEST_MONGO_URI=mongodb://root:prisma@mongo:27018/tests?authSource=admin
       - TEST_MONGO_URI_MIGRATE=mongodb://root:prisma@mongodb_migrate:27017/tests-migrate?authSource=admin
       # CockroachDB
-      - TEST_COCKROACH_URI=postgresql://prisma@cockroachdb:26257/
+      - TEST_COCKROACH_URI=postgresql://prisma@localhost:26257/tests
```

```suggestion
      - TEST_COCKROACH_URI=postgresql://prisma@cockroachdb:26257/tests
```
Host needs to be `cockroachdb` in Buildkite (name of the docker-compose step)